src/main/java/org/apache/sling/security/impl/ReferrerFilter.java

```diff
@@ -430,6 +448,22 @@ private boolean isValidRegexReferrer(HostInfo hostInfo) {
         return false;
     }
 
+    /**
+     * Returns <code>true</code> if the path info associated with the given request is contained in the configured excluded paths.
+     *
+     * @param request The request to check
+     * @return <code>true</code> if the path-info associate with the given request is contained in the configured excluded paths.
+     */
+    private boolean isExcludedPath(HttpServletRequest request) {
+        String path = request.getPathInfo();
+        for (final String excludedPath : this.excludedPaths) {
+            if (excludedPath.equals(path)) {
```

Comment:
I would add a null check for excludedPath here, as a sanity check

Reply:
sure -> done

Reply:
ah sorry, I meant "excludedPath" :) so a string in the array might be null

Reply:
but checking excludedPaths for null is good as well

Reply:
:-)

Reply:
sodele

Reply:
Thanks